app/client/src/apis/getImgURL.ts

```
@@ -0,0 +1,22 @@
+
+export async function getImgURL(file: string) {
+  const formData = new FormData();
+  formData.append("image", file);
+
+  const req = {
+    formData,
+    Value: file,
+    Key: "1image",
+  };
+
+  const res = await API<Object>("/api/v1/image", {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)